buildkite/scripts/rosetta-integration-tests.sh

```diff
@@ -232,8 +232,9 @@ for zkapp_path in ${ZKAPP_PATH}/*/; do
   }
 }
 EOF
+  curr_dir=$(pwd)
   cd "$zkapp_path"
-  npm install --no-progress src/lib/snarky_js_bindings/snarkyjs
+  npm install --no-progress ${curr_dir}/src/lib/snarky_js_bindings/snarkyjs
```

Comment:
Posix compliant shells will set $PWD variable for you. You can just use it instead of creating `curr_dir`
`npm install --no-progress ${PWD}/src/lib/snarky_js_bindings/snarkyjs`

Reply:
there's also $OLDPWD variable that is meant to be used for what you're describing

Reply:
That I didn't know about. Thanks for the tip!